roles/container-engine/containerd/tasks/main.yml

```diff
@@ -48,6 +48,8 @@
     - (ansible_distribution_major_version | int) >= 31
     - ansible_proc_cmdline['systemd.unified_cgroup_hierarchy'] is not defined or ansible_proc_cmdline['systemd.unified_cgroup_hierarchy'] != '0'
 
+- import_tasks: pre-upgrade.yml
```

Comment:
Note that with containerd 1.4.3 we could actually support cgroups v2 relatively easily. Just need to make sure that the `systemd` cgroupfs driver is used for `runc` and that's it